app/schemas/models/classroom.schema.coffee

```diff
@@ -39,6 +39,19 @@ _.extend ClassroomSchema.properties,
       name: {type: 'string'}
       slug: {type: 'string'}
       position: c.point2d()
+
+      # properties relevant for voyager campaigns 
```

Comment:
Should this be a description in the schema?

Reply:
Will add the specific descriptions.